src/components/unmute-indication/_unmute-indication.scss

```diff
@@ -1,3 +1,5 @@
+$color: 'white';
```

Comment:
no need to define new scss var for white, this is already defined in the _variables.scss

Reply:
its more for the color of the unmute button

Reply:
the color is white, so until this will be changed, no need for another definition IMO

Reply:
@OrenMe ?

Reply:
@dan-ziv as it is used in multiple places and we will need to support theming it's better to start putting such things as settings attribute - although it is not connected to anything ATM.
Also - setting a css param called white and then using it instead has no value over inlining it.

Reply:
@dan-ziv if there's no other issue please remove the change request. thanks.